cognite/client/data_classes/data_modeling/ids.py

```diff
@@ -193,7 +194,7 @@ def create_args(id_: Id) -> tuple[str, str, str | None, Literal["node", "edge"]
                 return id_[0], id_[1], None, id_type  # type: ignore[return-value]
             raise ValueError("Instance given as a tuple must have two elements (space, externalId)")
         if isinstance(id_, tuple):
-            return id_[0], id_[1], id_[2] if len(id_) == 3 else None, None
+            return id_[0], id_[1], (id_[2] if len(id_) == 3 else None), None
```

Comment:
Not sure if this is more readable or not, you choose 😜 
```suggestion
            return (*id_, None, None)[:4]
```

Reply:
Can we catch this in a test?

Reply:
(nvm i see it's already done 💯)

Reply:
@erlendvollset That change was only for easier readability